backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5574380_sys_gh10388_UpdateIndexToConsiderOrg.sql

```diff
@@ -0,0 +1,3 @@
+DROP INDEX c_elementvalue_value;
+CREATE UNIQUE INDEX c_elementvalue_value
+	ON c_elementvalue (c_element_id, ad_org_ID, value);
```

Comment:
have u also checked AD_Table.AccessLevel because it looks like we have to change it from Client to Client+Org ?

Reply:
Yes, you are right!